runtime-modules/content/src/permissions/curator_group.rs

```diff
@@ -1,4 +1,47 @@
 use super::*;
+use sp_std::collections::btree_map::BTreeMap;
+#[cfg(feature = "std")]
+use strum_macros::EnumIter;
+
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize, EnumIter))]
+#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, Debug, PartialOrd, Ord)]
+pub enum ChannelFeature {
```

Comment:
Would it perhaps be useful to write an explicit comment on each of these which explains exactly what extrinsics, with what parameter inputs, are off limits as a result of each variant? I was first a bit skeptical that the implementation was correct when reading `update_video`, because  I had some loose idea in my mind about these variants being mapped to explicit singular extrinsics, but they are not, as some features can be triggered by multiple extrinsics, such as issuing an NFT.

Reply:
Done in https://github.com/Joystream/joystream/pull/3366/commits/d164b4b310c6af9f0affc7a76abdbfdd29e77938